IdelPog/Assets/Scripts/IdelPog/Orchestration/Currency/ICurrencyMediator.cs

```diff
@@ -0,0 +1,27 @@
+﻿using IdelPog.Service;
+using IdelPog.Structures;
+
+namespace IdelPog.Orchestration.Currency
+{
+    /// <summary>
+    /// 
```

Comment:
Should remove this summary tag. Otherwise, fill it with some text if you think it would be benefical